libresapi/src/api/RsControlModule.cpp

```diff
@@ -125,6 +136,7 @@ void RsControlModule::run()
     bool login_ok = false;
     while(!login_ok)
     {
+		mPassword = "";

```

Comment:
this is not thread save. RsControlModule::run() runs in a different thread than the webserver thread.
So access to RsControlModule member variables must lock the mDataMtx first.
```C
{
    RsStackMutex stack(mDataMtx); // ********** LOCKED **********
    mPassword = "";
}
```